opus/application/static_media/js/widgets.js

```diff
@@ -93,18 +93,71 @@ var o_widgets = {
             }
         });
 
-        // close opened surfacegeo widget if user select another surfacegeo target
+        // Update surfacegeo widgets in place if user selects another surfacegeo target.
+        // 1. Update surfacegeo attributes/text in DOMs of all related surfacegeo widgets.
+        // 2. Update selections (opus.selection, opus.extras) & widgets.
         $('#search').on('change', 'input.singlechoice', function() {
```

Comment:
Don't we need some kind of test here that it's the surface geo selection that's changing? What if we had another radio button widget in the future?